hal/mbed_itm_api.c

```diff
@@ -56,22 +56,21 @@ void mbed_itm_init(void)
         ITM->TPR  = 0x0;
 
         /* Trace Control Register */
-        ITM->TCR  = (1 << ITM_TCR_TraceBusID_Pos) | 
-                    (1 << ITM_TCR_DWTENA_Pos)     | 
+        ITM->TCR  = (1 << ITM_TCR_TraceBusID_Pos) |
+                    (1 << ITM_TCR_DWTENA_Pos)     |
                     (1 << ITM_TCR_SYNCENA_Pos)    |
                     (1 << ITM_TCR_ITMENA_Pos);
 
         /* Trace Enable Register */
-        ITM->TER = SWO_STIMULUS_PORT;    
+        ITM->TER = SWO_STIMULUS_PORT;
     }
 }
 
 uint32_t mbed_itm_send(uint32_t port, uint32_t data)
 {
     /* Check if ITM and port is enabled */
     if (((ITM->TCR & ITM_TCR_ITMENA_Msk) != 0UL) &&      /* ITM enabled */
-        ((ITM->TER & (1UL << port)     ) != 0UL)   )     /* ITM Port enabled */
-    {
+            ((ITM->TER & (1UL << port)) != 0UL)) {           /* ITM Port enabled */
```

Comment:
This style change actually upsets me.

Imo, its much easier to read multiline conditional statements when the left parenthesis are aligned based on their parenthesis depth level. This change throws all of that out the window.

Reply:
That is the default behavior we use (no config in the astylerc, neither in our code style guide for this). Parameters are aligned if on multilines but conditions are indented.